xarray/tests/test_units.py

```diff
@@ -2089,28 +2079,28 @@ def test_pad(self, dtype):
             pytest.param(unit_registry.m, None, id="identical_unit"),
         ),
     )
-    def test_pad_with_fill_value(self, unit, error, dtype):
+    def test_pad_unit_constant_value(self, unit, error, dtype):
```

Comment:
I'm not sure we should do that, the only difference between both `pad` tests is the tested data. I realize that that's something that was already the case before this PR, so instead of touching the test above it might make sense to rename this test to `test_pad` and temporarily disable the other test by marking it with `skip`. I'll think a bit more about both tests and try to combine them in a new PR.